lib/appcontroller_client.py

```diff
@@ -367,6 +379,151 @@ def set_deployment_id(self, deployment_id):
     Returns:
       A boolean indicating whether the deployment ID is stored or not.
     """
-    return self.run_with_timeout(10, 'Set deployment ID request timed out.',
-      self.DEFAULT_NUM_RETRIES, self.server.set_deployment_id, self.secret,
-      deployment_id)
+    return self.run_with_timeout(self.DEFAULT_TIMEOUT, 'Set deployment ID request timed out.',
+      self.DEFAULT_NUM_RETRIES, self.server.set_deployment_id, self.secret, deployment_id)
+
+
+  def get_all_stats(self):
+    """ Queries the AppController for all the stats.
+
+    Returns:
+      A hash in string format containing system and platform stats for this node.
+    """
+    return self.run_with_timeout(self.LONGER_TIMEOUT, 'Get all JSON stats request timed out.',
+      self.DEFAULT_NUM_RETRIES, self.server.get_all_stats, self.secret)
+
+
+  def does_app_exist(self, appname):
+    """ Queries the UserAppServer to see if the named application exists,
```

Comment:
It should be the AppController. I will change that and the other comments with references to the UserAppServer.